src/inputType/RotatePanInput.ts

```diff
@@ -0,0 +1,136 @@
+import Axes from "../Axes";
+import { toAxis } from "./InputType";
+import { PanInput, PanInputOption } from "./PanInput";
+
+export class RotatePanInput extends PanInput {
+	private rotateOrigin: number[];
+	private prevAngle: number;
+	private prevQuadrant: number;
+	private lastDiff: number;
+	private coefficientForDistanceToAngle: number;
+
+	constructor(el: string | HTMLElement, options?: PanInputOption) {
+		super(el, options);
+
+		const rect = this.element.getBoundingClientRect();
+		// TODO: how to do if element is ellipse not circle.
+		this.coefficientForDistanceToAngle = 360 / (rect.width * Math.PI); // from 2*pi*r * x / 360
```

Comment:
Should it have to be different with circle and ellipse?
I think we can generate degree by center position, and calculating position.

Reply:
@WoodNeck 
`coefficentForDistanceToAngle` is for calculating coefficient to convert from circumference(by user input distance) to angle.

But the circumference would be different between circle and ellipse.  So I had commented it (for later support)